tests/e2e/triggers/workloads.yaml

```diff
@@ -18,14 +25,17 @@ workloads:
     args: ["--bolt-port", *bolt_port]
     proc: "tests/e2e/triggers/procedures/"
     <<: *template_cluster
-  - name: "ON DELETE Triggers"
+  - name: "ON DELETE Triggers Storage Properties On Edges True"
     binary: "tests/e2e/triggers/memgraph__e2e__triggers__on_delete"
     args: ["--bolt-port", *bolt_port]
     proc: "tests/e2e/triggers/procedures/"
     <<: *template_cluster
-  - name: "Triggers privilege check"
-    binary: "tests/e2e/triggers/memgraph__e2e__triggers__privileges"
-    args: ["--bolt-port", *bolt_port]
-    <<: *template_cluster
-
-
+  # - name: "Triggers privilege check"
+  #   binary: "tests/e2e/triggers/memgraph__e2e__triggers__privileges"
+  #   args: ["--bolt-port", *bolt_port]
+  #   <<: *template_cluster
```

Comment:
Please uncomment or remove.

Reply:
Will do on the last commit because I cannot run this test locally.